packages/web-app-files/src/helpers/resource/copyMove.ts

```diff
@@ -215,6 +215,14 @@ export const copy = async (
     true
   )
 }
+
+export const resolveFileNameDuplicate = (name, existingFiles, iteration = 1) => {
+  const potentialName = `${name} (${iteration})`
```

Comment:
this only works for folders, not for files, right?

Reply:
ah yes, right